node/src/main/scala/coop/rchain/node/api/DeployGrpcService.scala

```diff
@@ -79,9 +77,18 @@ private[api] object DeployGrpcService {
         )
       }
 
+      override def machineVerifiableDag(q: MachineVerifyQuery): Task[GrpcEither] =
+        defer(BlockAPI.machineVerifiableDag[F])
+
       override def showBlocks(request: BlocksQuery): Observable[GrpcEither] =
         Observable
-          .fromTask(deferList(BlockAPI.showBlocks[F](request.depth)))
+          .fromTask(
+            deferList[BlockInfoWithoutTuplespace](
+              Functor[F].map(BlockAPI.showBlocks[F](Some(request.depth)))(
```

Comment:
Weird that this hack is needed to make `map` work. Let me know when you find out why that is :)